### PR TITLE
docs(guidelines): clean up "frequent code reviewers" list

### PR DESCRIPTION
**Pull request by wincent**
-   Removing myself and Chema because we are less involved at this point.
-   Swapping in Matu, Javi de A, y Javi G for Jon Mak so that the list basically consists of all the leads of within Frontend Infrastructure; ~the only reason I am not adding Javi Arcos to this list at this time is that, at least for now, I imagine that the Headless squad will continue to use [their own user's fork](https://github.com/liferay-headless/liferay-portal/pulls).~ [Not true: we're going to consolidate]

    In theory, even if the tech leads aren't the right person for a given PR, they should be in a good position to triage it and assign it to somebody who is.

### CODE REVIEW

File: guidelines/dxp/code_review.md [hunk 1]
```diff
@@ -22,10 +22,9 @@ These are guidelines that the Frontend Infrastructure Team follows when developi
     Note that if you create PRs with [the `gh` command-line tool](https://www.npmjs.com/package/gh) you obviously can't see or check a checkbox. In that case, you may wish to add the people who most frequently review your code as [collaborators to your repo](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository), as well as the team members who tend to be generally involved in code review:
 
     -   [bryceosterhaus](https://github.com/bryceosterhaus)
-    -   [jbalsas](https://github.com/jbalsas)
-    -   [jonmak08](https://github.com/jonmak08)
+    -   [javierdearcos](https://github.com/javierdearcos)
     -   [markocikos](https://github.com/markocikos)
-    -   [wincent](https://github.com/wincent)
+    -   [matuzalemsteles](https://github.com/matuzalemsteles)
```
Comment (jbalsas) [inline guidelines/dxp/code_review.md:27]: shouldn't nhpatt be here too?

Comment (wincent) [reply guidelines/dxp/code_review.md:27]: I thought that if he was stepping into your role he might want to delegate. But if he wants to be in the list, he definitely can be!

`@nhpratt` 😂 — speak your mind!

**Comment by wincent** [comment]
Also mentioning @javierdearcos because he's not a collaborator in this repo (yet) so I can't mark him as a reviewer.

File: guidelines/dxp/code_review.md (hunk 1 above)
Comment (nhpatt) [reply guidelines/dxp/code_review.md:27]: Yeah, let me in! :D

Comment (jbalsas) [reply guidelines/dxp/code_review.md:27]: I knew @nhpatt wouldn't be able to resist the urge...

more

Comment (jbalsas) [reply guidelines/dxp/code_review.md:27]: > @nhpratt 😂 — speak your mind!

snap

Comment (nhpatt) [reply guidelines/dxp/code_review.md:27]: I'm going to change my nickname in twitter to @nhpratt just to have a mention by the great @wincent

**Comment by javierdearcos** [comment]
I am a collaborator now! Can I collaborate? please? please? please?

**Comment by wincent** [comment]
Lesson `#1` for the new additions to the Frontend Infra team:

**If you include a ninja gif in a PR comment, you are entitled to do a ninja merge.**

(Or at least, I think those are the rules because @jbalsas has been letting me get away with it for some time now.)

Witness:

ninja

**Comment by jbalsas** [comment]
> If you include a ninja gif in a PR comment, you are entitled to do a ninja merge.
>
> (Or at least, I think those are the rules because @jbalsas has been letting me get away with it for some time now.)

Sorry to disappoint, but if it's not written down on our guidelines it's not a thing. I just love ninja gifs, that's all :)

**Comment by wincent** [comment]
> Sorry to disappoint, but if it's not written down in our guidelines then it's not a thing.

breaking-my-heart

**Pull request merged**